libio/misc/misc.c

```diff
@@ -362,10 +362,10 @@ setup_corefile(void)
 }
 
 void
-libio_init(int daemonn)
+libio_init(int daemon)
 {
 #ifndef _WIN32
-  if (daemonn)
+  if (daemon)
```

Comment:
Fairly certain this is local -- I can't figure out what it's describing, it feels like a boolean (from here and the call site), so I can't see any particularly obvious reason for the extra `n` -- I really looked.